electronic_store/data/base_automation.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="base_automation_1" model="base.automation">
+        <field name="name">warranty date update</field>
+        <field name="model_id" ref="stock.model_stock_move_line"/>
+        <field name="filter_domain">["&amp;", ("state", "=", "done"), ("picking_code", "=", "outgoing")]</field>
+        <field name="trigger">on_create_or_write</field>
+        <field name="trigger_field_ids" eval="[(6, 0, [ref('stock.field_stock_move_line__state')])]"/>
```

Comment:
Does that mean the automated action will be called on all stock move line at any change in one those?